custom_components/frigate/sensor.py

```diff
@@ -408,3 +414,70 @@ def unit_of_measurement(self) -> str:
     def icon(self) -> str:
         """Return the icon of the sensor."""
         return self._icon
+
+
+class CoralTempSensor(FrigateEntity, CoordinatorEntity):  # type: ignore[misc]
+    """Frigate Coral Temperature Sensor class."""
+
+    _attr_entity_category = EntityCategory.DIAGNOSTIC
+
+    def __init__(
+        self,
+        coordinator: FrigateDataUpdateCoordinator,
+        config_entry: ConfigEntry,
+        name: str,
+    ) -> None:
+        """Construct a CoralTempSensor."""
+        self._name = name
+        FrigateEntity.__init__(self, config_entry)
+        CoordinatorEntity.__init__(self, coordinator)
+        self._attr_entity_registry_enabled_default = False
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID to use for this entity."""
+        return get_frigate_entity_unique_id(
+            self._config_entry.entry_id, "sensor_temp", self._name
```

Comment:
What is the name that Frigate returns here? 

If there are other temperatures we could imagine that we get from somewhere else, we may wish to say 'coral_sensor_temp' here to disambiguate, but if 'coral' is already in the name (and  there's enough to tell the devices apart) then maybe this is unnecessary. What do you think?

Reply:
It will be based off of the driver name, in this case it will be `apex_{0 based index}` and I imagine it would be something else for other devices

Reply:
no problem to add but right now I believe apex is unique to the coral

Reply:
OK -- then we should either rename this class to `DeviceTempSensor` or go the other direction and add `coral_` into the unique_id. I think the former is better, i.e. we'd just expose any device temperature information using this entity?